internal/handlers/sqlite/msg_collstats.go

```diff
@@ -76,6 +76,25 @@ func (h *Handler) MsgCollStats(ctx context.Context, msg *wire.OpMsg) (*wire.OpMs
 		return nil, lazyerrors.Error(err)
 	}
 
+	list, err := db.ListCollections(ctx, new(backends.ListCollectionsParams))
```

Comment:
So, here we call `ListCollection` only because we want to find the collection info of a particular collection.
I see just calling `db.Collection()` is not possible because it returns `Collection` and not `CollectionInfo`.

Did we consider an approach when we can perform something like `db.Collection(collection).Info()` to get `CollectionInfo` of a particular collection? Calling the whole list and iterating through it when we only need one collection doesn't look super reasonable.

Reply:
Why does that not look reasonable?

Reply:
Because inside `r.CollectionList` we iterate through all the collections in the DB (`r.colls`) to make a slice with response data, and then we iterate through this resulting slice once again (here) to find one particular collection by its name. Instead of that, we can just get the exact collection from the map  `r.colls` by its name.

Reply:
And add a method for every backend to implement that does not add anything new.

I don't think that iterating over memory is worse than that.